src/app/not-found.tsx

```diff
@@ -2,7 +2,7 @@
 
 import { PageNotFound } from "@/assets/images";
 import { Button } from "@/components/atoms";
-import { ROUTER } from "@/constants";
+import ROUTER from "@/constants/router.constant";
```

Comment:
이 ROUTER도 constants/index.ts에서 재내보내기 했으니 "@/constants" 에서 불러오는게 맞을 것 같습니다
```suggestion
import { ROUTER } from "@/constants";
```

Reply:
위와 같습니다!!